src/Widgets/BatteryBox.vala

```diff
@@ -8,7 +8,7 @@ public class Power.BatteryBox : Gtk.Grid {
         var battery_header = new Granite.HeaderLabel ("");
 
         var devices_box = new Gtk.Box (HORIZONTAL, 24) {
-            margin_bottom = 6
```

Comment:
This is possibly by design.

https://docs.elementary.io/hig/widgets/creating-layouts

> 
Horizontal spacing between buttons is 6px.